src/GeniusYield/Test/Privnet/Setup.hs

```diff
@@ -76,7 +76,7 @@ The first argument is the log severity filter. Only logs of this severity or hig
 newtype Setup = Setup (GYLogSeverity -> (String -> IO ()) -> (Ctx -> IO ()) -> IO ())
 
 cardanoDefaultTestnetOptionsConway :: CardanoTestnetOptions
-cardanoDefaultTestnetOptionsConway = cardanoDefaultTestnetOptions {cardanoNodeEra = Api.AnyCardanoEra Api.ConwayEra}
+cardanoDefaultTestnetOptionsConway = cardanoDefaultTestnetOptions {cardanoNodeEra = Api.AnyCardanoEra Api.ConwayEra, cardanoEpochLength = 2000}
```

Comment:
As noted in [this](https://github.com/geniusyield/atlas/issues/372#issuecomment-2551249498) comment, I think it's best to just increase epoch length.